crates/digilogic/src/ui/settings.rs

```diff
@@ -115,7 +115,14 @@ impl egui_dock::TabViewer for TabViewer<'_> {
 
     fn ui(&mut self, ui: &mut Ui, tab: &mut Self::Tab) {
         match *tab {
-            Page::Appearance => self.context.style_ui(ui),
+            Page::Appearance => {
+                let theme = if self.settings.dark_mode {
```

Comment:
If `AppSettings::dark_mode` where `egui::Theme`, then this would be more concise, but then you'd need a similar conditional for the global theme preferences switch elsewhere.